specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/syncGroups.json

```diff
@@ -0,0 +1,1083 @@
+          "x-ms-enum": {
+            "name": "SyncGroupLogType",
+            "modelAsString": true
+          }
+        },
+        "source": {
+          "description": "Source of the sync group log.",
+          "type": "string",
+          "readOnly": true
+        },
+        "details": {
+          "description": "Details of the sync group log.",
+          "type": "string",
+          "readOnly": true
+        },
+        "tracingId": {
+          "format": "uuid",
+          "description": "TracingId of the sync group log.",
+          "type": "string",
+          "readOnly": true
+        },
+        "operationStatus": {
+          "description": "OperationStatus of the sync group log.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SyncGroupProperties": {
+      "description": "Properties of a sync group.",
+      "type": "object",
+      "properties": {
+        "interval": {
+          "format": "int32",
+          "description": "Sync interval of the sync group.",
+          "type": "integer"
+        },
+        "lastSyncTime": {
+          "format": "date-time",
+          "description": "Last sync time of the sync group.",
+          "type": "string",
+          "readOnly": true
+        },
+        "conflictResolutionPolicy": {
+          "description": "Conflict resolution policy of the sync group.",
+          "enum": [
+            "HubWin",
+            "MemberWin"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SyncConflictResolutionPolicy",
+            "modelAsString": true
+          }
+        },
+        "syncDatabaseId": {
+          "description": "ARM resource id of the sync database in the sync group.",
+          "type": "string"
+        },
+        "hubDatabaseUserName": {
+          "description": "User name for the sync group hub database credential.",
+          "type": "string"
+        },
+        "hubDatabasePassword": {
+          "description": "Password for the sync group hub database credential.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
+        },
+        "syncState": {
+          "description": "Sync state of the sync group.",
+          "enum": [
+            "NotReady",
+            "Error",
+            "Warning",
+            "Progressing",
+            "Good"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "SyncGroupState",
+            "modelAsString": true
+          }
+        },
+        "schema": {
+          "$ref": "#/definitions/SyncGroupSchema",
+          "description": "Sync schema of the sync group."
+        },
+        "usePrivateLinkConnection": {
+          "description": "If use private link connection is enabled.",
+          "type": "boolean",
+          "x-ms-mutability": [
+            "create",
+            "update"
```

Comment:
Why can't this be read?

Reply:
synMembers swagger defined it as just boolean.

---
In reply to: [426190184](https://github.com/Azure/azure-rest-api-specs/pull/9454#discussion_r426190184) [](ancestors = 426190184)

Reply:
Good point, the swagger annotations look inconsistent. Changing both to be readable since I see no reason for it not to be.